glean-core/src/upload/policy.rs

```diff
@@ -9,6 +9,14 @@ const MAX_WAIT_ATTEMPTS: u32 = 3;
 const MAX_PING_BODY_SIZE: usize = 1024 * 1024; // 1 MB
 const MAX_PENDING_PINGS_DIRECTORY_SIZE: u64 = 10 * 1024 * 1024; // 10MB
 
+// The average number of baseline pings per client (on Fenix) is at 15 pings a day.
+// The P99 value is ~110.
+// With a maximum of (a nice round) 250 we can store about 2 days worth of pings.
```

Comment:
What is the average size of ping files? If Sebastian had more than 5k files and never hit the 10MB limit for ping folder, adding the 250 limit on the number of pings may make that max size almost obsolete, no?

I still haven't gotten to https://bugzilla.mozilla.org/show_bug.cgi?id=1657852, but that should give us some insight into this. 

We can re-evaluate that on a follow up, but I really think we should at some point. Maybe it's just remove that logic altogether.

Reply:
Uh, you're right. I should have done the calculations. The baseline pings I have here from my tests are ~600 bytes.
At 10 MB for the directory that's 16k pings, so yeah, we never reach that.

Now metrics and events pings are most likely bigger, a reevaluation after your analysis would be good.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1665653